### PR TITLE
EntityDataInterpreter allows to add extra validation via event

### PR DESCRIPTION
**Pull request by senaria**

### CODE REVIEW

File: Importer/Interpreter/EntityDataInterpreter.php [hunk 1]
```diff
@@ -273,6 +275,21 @@ protected function getEntity(array $interpretedData)
...
+     * @param object $entity
+     *
+     * @return ConstraintViolationListInterface
+     */
+    private function validate($entity)
+    {
+        $violationList = $this->validator->validate($entity);
+
+        $event = new PostImportValidationEvent($violationList, $entity);
+        $this->eventDispatcher->dispatch(ImportEvents::POST_VALIDATION, $event);
```
Comment (mkruk-u2) [inline Importer/Interpreter/EntityDataInterpreter.php]: So if any listener of this event can actually modify list of validation errors, it's not really a "post" validation event, it's still a part of validation process.

File: Importer/Event/ImportEvents.php [hunk 2]
```diff
@@ -8,4 +8,6 @@
     const POST_FIELD_INTERPRET = 'EVENT_POST_FIELD_INTERPRET';
 
     const PRE_BIND_DATA = 'EVENT_PRE_BIND_DATA';
+
+    const POST_VALIDATION = 'EVENT_POST_VALIDATION';
```
Comment (yakobe) [inline Importer/Event/ImportEvents.php]: the order of this file is a bit strange. it seems it _was_ alphabetical but now its random. perhaps we could make it into the order they happen?

**Comment by yakobe** [comment]
I'm not sure about the reasoning behind this change. What are you trying to achieve? It seems you want to add to the validation. Perhaps this could be solved in a better way?

File: Importer/Event/ImportEvents.php (hunk 2 above)
Comment (senaria) [inline Importer/Event/ImportEvents.php]: ok

File: Importer/Interpreter/EntityDataInterpreter.php (hunk 1 above)
Comment (senaria) [inline Importer/Interpreter/EntityDataInterpreter.php]: yeha like that approach. Than it is on validation. Modify is a wide term, lets they they can add new ones. I will have a though.

**Pull request closed without merging**